### PR TITLE
create_lxc: resolve test vm not exist problem

### PR DESCRIPTION
**Pull request by shuangqian**
Modify the vms = '' and main_vm = "lxc_test_vm1" in the cfg file.

Modify the virsh_create_lxc.py file to get vm_name from "main_vm",
and change the way to check the result of the virsh.create().

### CODE REVIEW

**Comment by waynesun09** [comment]
Please rebase the PR since you have 2 commits with same description

File: libvirt/tests/src/virsh_cmd/domain/virsh_create_lxc.py [hunk 1]
```diff
@@ -90,17 +104,24 @@ def container_xml_generator():
...
+        match = re.search(r"lxc_test_vm1", list_result.stdout)
+        if match:
+            get_list_vmname = match.group()
+        else:
+            get_list_vmname = None
+
         if "--autodestroy" in options:
-            if vm.is_alive():
+            if check_state("running"):
                 raise error.TestFail("Guest still exist after close session "
```
Comment (waynesun09) [inline libvirt/tests/src/virsh_cmd/domain/virsh_create_lxc.py]: Before TestFail, you need destroy vm to clean env

Maybe you only need use check_state once, and then check the other options here.

File: libvirt/tests/src/virsh_cmd/domain/virsh_create_lxc.py [hunk 2]
```diff
@@ -90,16 +104,28 @@ def container_xml_generator():
                 logging.info("Find open file in guest: %s", cmd_output)
 
         session.close()
-        vm = env.get_vm(vm_name)
-        if "--autodestroy" in options:
-            if vm.is_alive():
+
+        list_result = virsh.dom_list()
+        match = re.search(r"lxc_test_vm1", list_result.stdout)
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/domain/virsh_create_lxc.py]: We should not search string "lxc_test_vm1" here as vm name is get through cfg

**Comment by will-Do** [comment]
I'm going to close this PR as the author no longer working on this project.

**Comment by will-Do** [comment]
[New version of https://github.com/autotest/tp-libvirt/pull/924](https://trello.com/c/tPH5OCP4/5-new-version-of-https-github-com-autotest-tp-libvirt-pull-924)

**Pull request closed without merging**